languages/python/exercises/concept/dicts/.meta/example.py

```diff
@@ -0,0 +1,21 @@
+def create_inventory(items):
+    inventory = dict()
+    for item in items:
+        inventory.setdefault(item, 0)
+        inventory[item] += 1
+    return(inventory)
```

Comment:
`return` is a keyword, not a function (although the Python interpreter will _allow_ you to use it a one. The standard use of `return` is as follows:
```suggestion
    return inventory
```

Reply:
Also, why not re-use the add_items function here?